src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/mobile.json

```diff
@@ -779,8 +779,8 @@
       }
     }
   },
-  "breadcrumbs-height-compact": {
-    "value": "{component-height-200}",
+  "🚫 breadcrumbs-height-compact": {
+    "value": "{component-height-100}",
```

Comment:
@GarthDB @lynnhao @nabuhasan I really wonder, if it is okay, that we deprecate a token and change the value in one step? Doesn't that lead to a breaking change?